flink-streaming-java/src/test/java/org/apache/flink/streaming/util/AbstractStreamOperatorTestHarness.java

```diff
@@ -286,13 +334,12 @@ public void setup(TypeSerializer<OUT> outputSerializer) {
 				createStreamTaskStateManager(environment, stateBackend, processingTimeService);
 			mockTask.setStreamTaskStateInitializer(streamTaskStateInitializer);
 
-			if (operator instanceof SetupableStreamOperator) {
+			if (operator == null) {
```

Comment:
This changes are overwriting lines introduced in the previous commit, so something is not quite right here.

Shouldn't some or all of the changes from this commit, that are touching TestHanresses, be squashed/moved to the previous commit?